app/models/application.rb

```diff
@@ -0,0 +1,9 @@
+class Application < ApplicationRecord
```

Comment:
How is an application form populated with questions? Do we need something like 
`has_many :questions, through: :answers`
though that admittedly doesn't look right...

Reply:
I would go through jobs. because the jobs have the questions associated with them? it's a two step process either way I suppose